public/js/app/views/BookmarkletView.js

```diff
@@ -6,6 +6,18 @@ define(["app/app",
     templateName: 'bookmarklet',
     template: Ember.Handlebars.compile(tpl),
 
+    didInsertElement: function() {
+      var field = this.$("#bookmarklet-recipients")
+      field.select2({ width: '100%' })
+
+      // When the select2 became multiline it change bookmarklet height, but we can't
+      // handle this via CSS rules, so use JS to increase iframe size accordingly
+      // Only way to interact with the scripts outside the iframe is postMessage
+      field.on('change', function() {
+        window.parent.postMessage(window.document.documentElement.offsetHeight, '*')
```

Comment:
Well, there will be no more "next dropdown change" once external website suddenly passes `0`.